### PR TITLE
tagging GeometricalPredicates and VoronoiDelaunay

### PR DESCRIPTION
**Pull request by skariel**

### CODE REVIEW

**Comment by tkelman** [comment]
Can you push the git tags so they show up under https://github.com/JuliaGeometry/VoronoiDelaunay.jl/releases and https://github.com/JuliaGeometry/GeometricalPredicates.jl/releases ?

File: VoronoiDelaunay/versions/0.0.3/sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+6c0ec6f0f1cf18db076e77f387c8959cacba2338
```
Comment (tkelman) [inline VoronoiDelaunay/versions/0.0.3/sha1]: this commit has not been pushed to the repository https://github.com/JuliaGeometry/VoronoiDelaunay.jl/commit/6c0ec6f0f1cf18db076e77f387c8959cacba2338

Comment (skariel) [inline VoronoiDelaunay/versions/0.0.3/sha1]: @tkelman done now!

**Comment by skariel** [comment]
@tkelman I tagged the new releases, and pushed VoronoiDelaunay 0.0.3

**Pull request merged**